docs/Tracing.md

```diff
@@ -40,11 +40,15 @@ Instrumenting or tracing execution can have overheads that may not be tolerated
 * At the compile stage the **autoInstrument** flag in the CompilationOptions structure enables instrumenting the compile of a network so that it is able to produce per operator Trace Events. If this flag is not enabled, there is no expectation of events at that level of detail (coarser grained events may still be produced).
 * At the execution stage the **ExecutionContext** may contain a **TraceContext** member describing what level of Trace Event verbosity is requested for this run. At the time of writing there are five verbosity levels defined:
     * `NONE` - no Trace Events are required and none should be emitted.
+    * `REQUEST`- only high level events relating to requests should be emitted (e.g. request beginning and end().
```

Comment:
typos: (e.g. ... )

Reply:
What's the typo here?

Reply:
(e.g. request beginning and end().

check the parentheses.